javascript/TreeDropdownField.js

```diff
@@ -154,17 +154,21 @@
 				var updateFn = function() {
 					var val = self.getValue();
 					if(val) {
-						
+
 						var node = tree.find('*[data-id="' + val + '"]'),
 							title = node.children('a').find("span.jstree_pageicon")?node.children('a').find("span.item").html():null;
 						if(!title) title=(node.length > 0) ? tree.jstree('get_text', node[0]) : null;
-						
+
 						if(title) {
 							self.setTitle(title);
 							self.data('title', title);
 						}
 						if(node) tree.jstree('select_node', node);
 					}
+					else {
+						self.setTitle(self.data('empty-title'));
```

Comment:
Either will work, if you do camelCase then jQuery will look for the camelCase then the dash-separated names. So it's actually more performant this way

The native JS data API requires camelCase, though.

I can change it, but it's ultimately not going to change the behaviour and be fractionally slower